### PR TITLE
Verify special edits

### PR DESCRIPTION
**Pull request by doelleri**
For #166 and #167

### CODE REVIEW

File: app/partials/errorDetail-Q029.html [hunk 1]
```diff
@@ -1,43 +1,41 @@
...
-    </table>
+<table id="Q029" class="error-detail" ng-if="editError">
+    <thead>
+        <tr>
+            <th scope="col" class="id">LAR Number</th>
+            <th scope="col" class="state_code">Reported State Code</th>
+            <th scope="col" class="county_code">Reported County Code</th>
+            <th scope="col" class="census_tract">Reported Census Tract</th>
+            <th scope="col" class="recommended">Recommended MSA/MD</th>
+            <td><select ng-model="selectedAnswer" ng-change="selectAll(selectedAnswer)"><option value="0">Select...</option><option value="1">All of the listed MSA/MD's should be updated</option><option value="2">None of the listed MSA/MD's should be updated</option></select></td>
```
Comment (poorgeek) [inline app/partials/errorDetail-Q029.html]: Need a label for this select. You should be able to add something like:
```html
<label for="selectAll" class="u-visually-hidden">Verify all MSA/MDs as</label>
<select id="selectAll">...</select>
```

File: app/scripts/controllers/specialErrorDetail.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    }, function(isLastPage) {
+        if (isLastPage) {
+            $scope.canVerify = true;
+        }
+    });
+
+    $scope.pageSize = $scope.pageSize || 10;
+    $scope.currentPage = $scope.currentPage || 1;
+
+    $scope.start = function() {
```
Comment (poorgeek) [inline app/scripts/controllers/specialErrorDetail.js]: Boo. Was hoping you were going to refactor the pagination out to its own directive :(

Don't worry about it now. We'll still merge this in.

Comment (doelleri) [inline app/scripts/controllers/specialErrorDetail.js]: I was just following your lead from the IRS report ;)

But yes, agreed on a refactor, just didn't want to hold the rest of this back for it.

File: app/partials/errorDetail-Q029.html [hunk 3]
```diff
@@ -1,43 +1,41 @@
...
+        </tr>
+    </thead>
+    <tbody>
+        <tr ng-repeat="item in editError.errors | paginate:start():end()">
+            <th scope="row" class="id"><label for="lar-{{item.properties['LAR number']}}">{{item.properties['LAR number']}}</label></th>
+            <td class="state_code">{{item.properties['Reported State Code']}}</td>
+            <td class="county_code">{{item.properties['Reported County Code']}}</td>
+            <td class="census_tract">{{item.properties['Reported Census Tract']}}</td>
+            <td class="recommended">{{item.properties['Recommended MSA/MD']}}</td>
+            <td><select id="lar-{{$index}}" ng-model="selects[start()+$index]"><option value="0">Select...</option><option value="1">Yes, the listed MSA/MD should be updated</option><option value="2">No, the listed MSA/MD should not be updated</option></select></td>
```
Comment (poorgeek) [inline app/partials/errorDetail-Q029.html]: Oops! This select should actually be labeled by the Recommended MSA/MD value. Can you please move the label from the LAR number to the MSA/MD?

File: app/partials/errorDetail-Q029.html (hunk 1 above)
Comment (doelleri) [inline app/partials/errorDetail-Q029.html]: Done in 8f5025c

File: app/partials/errorDetail-Q029.html (hunk 3 above)
Comment (doelleri) [inline app/partials/errorDetail-Q029.html]: Done in 2ad70c2

**Pull request merged**